crates/ra_ide/src/diagnostics.rs

```diff
@@ -397,7 +397,7 @@ mod tests {
                 x <|>/ y
             }
 
-            //- /std/lib.rs
+            //- /core/lib.rs
             pub mod string {
                 pub struct String { }
             }
```

Comment:
This and other `String` tests look odd now. It seems like we can just replace `String` with `()` in all cases  in tests? 

Otherwise, r=me